base/docs/Docs.jl

```diff
@@ -291,14 +291,15 @@ end
 
 uncurly(@nospecialize ex) = isexpr(ex, :curly) ? ex.args[1] : ex
 
-namify(@nospecialize x) = astname(x, isexpr(x, :macro))
+namify(@nospecialize x) = astname(x, isexpr(x, :macro))::Union{Symbol,Expr,GlobalRef}
```

Comment:
This is the one I'm least certain about. Anyone who knows the docsystem care to comment on whether this is safe? @MichaelHatherly?

Reply:
Yeah, pretty sure that should be fine.

Reply:
That's a relief, thanks so much for checking!